scst-scan/bring-your-own-scanner.hbs.md

```diff
@@ -16,3 +16,17 @@ You might have an existing investment in a scan solution that VMware does not ha
 1. [Create an ImageVulnerabilityScan](ivs-create-your-own.hbs.md): Create an ImageVulnerabilityScan template that tells the Tanzu Application Platform how to run your scanner.
 2. [Verify your ImageVulnerabilityScan](verify-app-scanning.hbs.md): Verify that your ImageVulnerabilityScan is working correctly so that downstream Tanzu Application Platform services work correctly.
 3. [Wrap your ImageVulnerabilityScan in a ClusterImageTemplate](clusterimagetemplates.hbs.md): The ClusterImageTemplate wraps the ImageVulnerabilityScan and allows the Tanzu Application Platform supply chain to run the scan job.
+
+## <a id="prerequisities"></a>Prerequisites
+
+TAP users will need to:
+
+- Provide their own Vulnerability Scanner Image either by:
+  - Using a publicly available image with the scanner CLI.
+    - For example, this image with Trivy scanner CLI from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
+  - Building their own image with the scanner CLI which allows for:
+    - More customizable scanning experience.
+      - For example, creating an image with the scanner CLI along with any required dependencies to run the scanner CLI.
+    - Managing vulnerabilities within the TAP user's custom image.
```

Comment:
Can you elaborate on this a bit?  I'm not sure what you are referring to.  Does this mean patching vulnerabilities that exist in it?

Reply:
Yes I meant patching vulnerabilities that existing the image which was an issue for us in Scan 1.0. But if the TAP user's build their own image, they would have more control on addressing the vulnerabilities that exist in it.

Reply:
I removed this line as requested.